scspell/__init__.py

```diff
@@ -893,6 +906,9 @@ def main():
     if args.debug:
         set_verbosity(VERBOSITY_MAX)
 
+    if len(args.ext) == 0:
```

Comment:
Use `if not args.ext`.

```suggestion
    if not args.ext:
```

Reply:
Done!